src/yargs.ts

```diff
@@ -56,6 +57,34 @@ const commandArgs = yargs
             }
         },
     })
+    .command({
+        command: 'cdn',
+        describe: 'Add or update the CDN support configuration',
+        builder(theYargs) {
+            return theYargs.option('theia', {
+                describe: 'Base URL of the CDN that will host Theia files',
+                requiresArg: true,
+                type: 'string',
+                default: 'https://cdn.jsdelivr.net/gh/davidfestal/che-theia-cdn@latest/che-theia-editor/',
```

Comment:
constants ? Also it should be delegated to a cdn class. Yargs should be very minimal in size